core/steam_client.gd

```diff
@@ -276,6 +338,61 @@ func save_steamcmd_session() -> void:
 	logger.debug("Successfully saved steamcmd session to", tokens_save_path)
 
 
+## Save the given accounts (e.g. {"shadowapex": {"SteamID": 1234}}) as an entry
+## in $STEAM_ROOT/config/loginusers.vdf
+func save_loginusers(accounts: Dictionary) -> void:
+	var users := {
+		"users": {}
+	}
+
+	# Load the existing loginusers if it exists
+	if FileAccess.file_exists(vdf_loginusers_path):
+		logger.debug(vdf_loginusers_path, "exists, loading existing configuration")
+		var loginusers_vdf := FileAccess.get_file_as_string(vdf_loginusers_path)
+		if loginusers_vdf.is_empty():
+			logger.warn("loginusers.vdf at", loginusers_vdf, "is empty. Unable to save login session.")
+		var vdf := Vdf.new()
+		if vdf.parse(loginusers_vdf) != OK:
+			logger.warn("Failed to parse", vdf_config_path, ":", vdf.get_error_message())
+		else:
+			users = vdf.data
+
+	for username in accounts.keys():
```

Comment:
Are there two formats?

Reply:
I'm not sure what you mean here. Could you elaborate?